opencti-platform/opencti-front/src/private/components/settings/groups/Group.tsx

```diff
@@ -344,6 +407,64 @@ const Group = ({ groupData }: { groupData: Group_group$key }) => {
               </Grid>
             </Paper>
           </Grid>
+          <Grid item={true} xs={12} style={{ marginTop: 30 }}>
+            <Typography
```

Comment:
I think, it will be good to have a dedicated component for triggers and digests to be mutualize with code in User.tsx component